docs/reference/team-processes/triage-process.md

```diff
@@ -125,13 +125,24 @@ You'll see an additional commit from "Bananafox" on Dependabot PRs. Bananafox is
 Keep in mind that Dependabot duty is shared between the FxA triage owner and the SubPlat triage owner. Try to move these PRs along at least once or twice a week, but also be sure to timebox yourself.
 :::
 
-If CI is green and the files changed look correct, approve the Dependabot PR and use the “squash and merge” option to combine bot commits so the package upgrades are easier to revert if problems arise. If you need Dependabot to rebase the change, comment on the PR with `@dependabot recreate`. We can't use `@dependabot rebase` because the PR is modified by Bananafox.
+#### Dependabot PRs that Pass CI
+
+Generally speaking, if CI is green, the dependency is very likely fine to merge. However, also take a look at the number shown on the "checks" tab of the PR. If there's less than 4 checks, it might actually cause a problem somewhere in the codebase, and if there's not already an issue filed for that package, you may want to ask the team if anyone thinks it may not be safe to merge.
```

Comment:
Probably should also skim the release notes about breaking changes.  Some packages are good about that.

Reply:
Thanks, added!